Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```
@@ -20,13 +20,12 @@ configuration:
     | eval rule_title=if(isnull(rule_title),rule_name,rule_title) | `get_urgency`
     | `risk_correlation` | eval rule_description=if(isnull(rule_description),source,rule_description)
     | eval security_domain=if(isnull(security_domain),source,security_domain)
-  display: Fetch notable events ES query
+  display: Fetch events query
   name: fetchQuery
   required: false
   type: 0
-  additionalinfo: The query for the notable events enrichment (defined by default).
-    If you decide to edit this, make sure to provide a query that uses the \`notable\`
-    macro. Use the new default query as an example.
+  additionalinfo: You can edit this query to fetch other types of events. 
+    Note that to fetch notable events, make sure the query uses the \`notable\` macro.
```

This line is unnecessary in my opinion, they are aware of it and we're not going to teach them SPL in my opinion

---

Alternatively, I would write something like: "The Splunk search query to fetch events by", every Splunk user would understand

---

We need to make sure that new customers are using the notable macro to build their environment (preprocess, pbs, etc...) with the notable macro. So I think we should keep this.
```suggestion
    Note that to fetch ES notable events, make sure to include the \`notable\` macro in your query.
```